doc/source/introduction_intro.rst

```diff
@@ -14,49 +14,93 @@ to facilitate access to and exchange of genomics data across remote sites.
 --------------------------
-For more details on web APIs, see `this wikipedia page <https://en.wikipedia.org/wiki/Web_API>`_  
 
-ReST protocols for data transfer are described `here <https://en.wikipedia.org/wiki/Representational_state_transfer>`_
+This API was created to enable researchers to better access and exchange genomic data across remote sites. Instead of having to
+download large BAM files or whole genome annotations, the API allows you to retrieve data about, for instance, single genes or genomic
+regions.  The API is designed for a variety of uses, from interactive visualization to large-scale analysis.
+
+For a full list of the GA4GH API goals, see :ref:`apigoals`
+
+The API consists of a series of :ref:`apidefinition`, or schemas, that define datasets, metadata, read group sets, reads, variants, etc. 
+
+The GA4GH web API schemas show developers how to connect data consumers ('clients') to data providers ('servers').
+They define how the data is organized, and thereby give information on what can be requested.
+The schemas are defined in Avro Interactive Data Language (extension .avdl), and 
+can be used to create code in any programming language.
 
 --------------------------
-The GA4GH web API
+API design patterns
 --------------------------
-This API was created to enable researchers to better access and exchange genomic data across remote sites. Instead of downloading complete BAM files or
-whole genome annotations, the API allows retrieval of information on, for instance, single genes or genomic regions.
+Web APIs use the protocols and conventions of the Web to connect consumers and providers of data, known as clients and servers.  This API uses these patterns:
 
-For a full list of the GA4GH API goals, see :ref:`apigoals`
+* HTTP protocol
+* ReST (Representational State Transfer) conventions
+* Paging
+* AVRO definition language
+* JSON representation
 
-The API consists of a series of :ref:`apidefinition`, or schemas, that define datasets, metadata, read group sets, reads, variants, etc. 
+For more details on web APIs, see `this wikipedia page <https://en.wikipedia.org/wiki/Web_API>`_  
 
-The schemas are written in Avro Interactive Data Language (extension .avdl). 
+HTTP protocol
+.............
+API clients use HTTP (`Hypertext Transfer Protocol <https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol>`_) for sending API queries and getting responses.
+HTTP is the same protocol that web browsers use for requesting pages from web servers.
 
-:ref:`avro` is a data serialization system, it defines how data gets transported across the internet.
-Here, data means both the request ('send me RNASeq for gene X in sample Y') and the response (the BAM file). Apache Avro only sends data in
-:ref:`json` or Binary Avro format, so requests and responses must be converted into one of those formats.
+In a typical conversation, a client sends a GET request to a server to ask for a specific resource, such as a set of reads.
```

Comment:
More typically in the current GA4GH API, clients send POST requests even to do "GET"ish things.  (They POST binary request objects.)